UnityProject/Assets/Scripts/UI/Tooltip.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+public class Tooltip : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler
+{
+    private static GameObject tooltipResource = null;
+    const float TOOLTIP_INTERVAL = 1.0f;
+    private float enterTime = 0;
+    private GameObject tooltipObject = null;
+    [SerializeField]
+    private string tooltipText = "";
+
+    void Start()
+    {
+        if (tooltipResource == null) {
+            tooltipResource = Resources.Load<GameObject>("UI/GUI/Tooltip");
```

Comment:
dont load with strings, just use the references in the editor

Reply:
But its always the same one, should you always drag the same component there?

Reply:
Yep make toolTipResource public or make it private and put the [Serializable] tag above it then you can drag and drop the Tooltop prefab into the Tooltip component on the inspector. Then Apply changes to the UI prefab or whatever prefab it is nested in and commit that.